src/screens/Learning.js

```diff
@@ -132,31 +127,31 @@ export default ({
             />
           </View>
           <View style={styles.heading}>
-            <SectionHeading text="Category: " />
+            <SectionHeading text={nativeLanguage === LANGUAGE_NAMES.EN ? "Category: " : "Sokajy :"} />
```

Comment:
create a file src/translations/index.js

```
export const CATEGORY_HEADING = 'CATEGORY_HEADING';


export const LANG_DATA = {
    [CATEGORY_HEADING]: {
    [LANGUAGE_NAMES.EN]: 'Category:',
    [LANGUAGE_NAMES.MG]: 'Sokajy: ',
  },
  
};
```

then you can get the translation like this

`const categoryHeading = LANG_DATA[CATEGORY_HEADING][nativeLanguage];`

then 
 ` <SectionHeading text={categoryHeading} />`